package.json

```diff
@@ -4,14 +4,14 @@
   "description": ":wave: Welcome to United Nepali!",
   "main": "index.js",
   "scripts": {
-    "postinstall": "cd server && npm install && cd ../client && npm install",
-    "start": "npm-run-all -p start:server start:client",
-    "start:server": "cd server && npm run start",
-    "start:client": "cd client && npm run start",
+    "start": "lerna bootstrap && lerna run start --stream",
+    "start-server": "lerna run start --scope=united-nepali-server",
+    "start-client": "lerna run start --scope=united-nepali-client",
     "cy:open": "npx cypress open",
     "cy:run": "npx cypress run",
-    "test": "start-server-and-test start 3000|8080 cy:open",
-    "test:run": "start-server-and-test start \"3000|8080\" cy:run"
+    "test": "start-server-and-test start http://localhost:3000 cy:open",
+    "test-run": "start-server-and-test start http://localhost:3000 cy:run",
```

Comment:
Was there a reason for removing `3000|8080`? Port 8080 is the client

Reply:
typo, i forgot to add it back after testing it